x-pack/plugins/cases/public/components/recent_cases/index.test.tsx

```diff
@@ -68,43 +66,50 @@ describe('RecentCases', () => {
     expect(getAllByTestId('case-details-link')).toHaveLength(7);
   });
 
-  it('is good at rendering max cases', () => {
+  it('render max cases correctly', () => {
     appMockRender.render(
       <TestProviders>
         <RecentCases {...{ ...defaultProps, maxCasesToShow: 2 }} />
       </TestProviders>
     );
-    expect(useGetCasesMock).toBeCalledWith({
-      initialQueryParams: { perPage: 2 },
+    expect(useGetCasesMock).toHaveBeenCalledWith({
+      filterOptions: { reporters: [] },
+      queryParams: { perPage: 2 },
     });
   });
 
-  it('updates filters', () => {
+  it('sets the reporter filters correctly', () => {
     const { getByTestId } = appMockRender.render(
       <TestProviders>
         <RecentCases {...defaultProps} />
       </TestProviders>
     );
 
-    const element = getByTestId('myRecentlyReported');
-    userEvent.click(element);
-    expect(setFilters).toHaveBeenCalled();
-  });
-
-  it('it resets the reporters when changing from my recently reported cases to recent cases', () => {
-    const { getByTestId } = appMockRender.render(
-      <TestProviders>
-        <RecentCases {...defaultProps} />
-      </TestProviders>
-    );
+    expect(useGetCasesMock).toHaveBeenCalledWith({
```

Comment:
Should we keep the behavior of the test and click on the `myRecentlyReported` tab?

Reply:
Never mind. I just saw that you combined the tests.